docs/api-specs/carboneio-api-spec.yaml

```diff
@@ -0,0 +1,426 @@
+openapi: 3.0.3
+
+info:
+  description: |-
+    Carbone Cloud/On-premise Open API reference.
```

Comment:
This is only for internal reference, as our api is working as a gateway in front of these endpoints, right? (if so, maybe clarify this here - or even move this out of the "docs" folder and into the export module only?)

We should also create a simple OpenAPI spec for our endpoints.